security-oauth2/src/main/java/io/micronaut/security/oauth2/grants/RefreshTokenGrant.java

```diff
@@ -32,7 +35,11 @@
  */
 @Introspected
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
-public class RefreshTokenGrant {
+public class RefreshTokenGrant extends AbstractSecureGrant implements AsMap {
```

Comment:
Where does the spec say that the refresh token grant needs secured?

Reply:
https://tools.ietf.org/html/rfc6749#section-6
>    Because refresh tokens are typically long-lasting credentials used to
   request additional access tokens, the refresh token is bound to the
   client to which it was issued.  If the client type is confidential or
   the client was issued client credentials (or assigned other
   authentication requirements), the client MUST authenticate with the
   authorization server as described in Section 3.2.1.